examples/exponential_modeling.ipynb

```diff
@@ -37,7 +37,17 @@
     "import seaborn as sns\n",
     "sns.set()\n",
     "\n",
-    "df = pd.read_csv(f'https://storage.googleapis.com/covid19-open-data/v2/{data_key}/main.csv').set_index('date')"
+    # ${base_url}/v3/aggregated.csv.gz
+    # .../v3/epidemiology.csv
+    # .../v3/demographics.csv
+    # .../v3/latest/aggregated.csv.gz <- Not ready yet!!!
+    # .../v3/latest/epidemiology.csv <- 1 row per location_key
+
+    # .../v2/${location_key}/main.csv
+    # .../v3/location/${location_key}.csv
+    # go/open-covid-data -> GCS Bucket where the data lives
+    #"df = pd.read_csv(f'https://storage.googleapis.com/covid19-open-data/v2/{data_key}/main.csv').set_index('date')"
```

Comment:
This is not valid formatting of ipynb files, how did you modify these?

Reply:
...by writing software like a hardware engineer. Fail. :( Thanks for the feedback! Re-factoring. :D